script/hypercube/hypercube.sh

```diff
@@ -196,13 +196,6 @@ function find_hypercubefile() {
   # using systemd services but meh I'm lazy and not very expert
   sleep 10
 
-  info "Install some dependencies..."
-  rm -rf /var/lib/apt/lists/*
-  apt-get clean &>> $log_file
-  apt-get update &>> $log_file
-
-  apt-get install -o Dpkg::Options::='--force-confold' -y --force-yes file udisks2 udiskie ntfs-3g jq  &>> $log_file || true
```

Comment:
* Removing this line depends on the this patch https://github.com/keomabrun/arm-images/commit/a3e7dbd96aee284b92ed2613603f9b21aeadf54d

So we can't merge this right now